portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,89 @@
+        Whether it was gaming or helping my family with their devices by troubleshooting and finding
+        the issue, it was always my niche.
+        <br>
+        </p>
+    <div>
+        <p> 
+        I am now attending the California State University of Northridge and am pursuing a
+        degree in Computer Information Technology and a minor in Geographic Information Science.
+        In the future with my degree, I hope to become a Data Scientist. Another occupation that 
+        I am also highly interested in becoming is becoming a Cyber Security Analyst.
+        </p>
+    </div>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <h2>My LinkedIn and GitHub</h2>
+    </div>
+        <p> https://github.com/mike39608/Coding-Portfolio 
+        <br> https://www.linkedin.com/in/michael-peters-4686b1227/
+        </p>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <h2>My Interests</h2>
+    </div>
+        <p> 
+            When it comes to what I like, I love to play sports and video games.
+            As well as watching shows, movies and anime. Here are some I am a big fan of.
+            I am currently learning to cook becuase the hobby had always interested me. watching
+            the food channnel and social media accounts propelled my interest into the world of
+            learning to cook foods. 
+            Other hobbies and skills I would love to learn and practice would be learning a new language
+            such as spanish or playing the piano.
+        </p>
+        <br>
+        <img alt="An Anime Game called 'Under Night'" src="/images/under night.jpg" width="400" height="300">
+        <img alt="An Anime called'Fire Force'" src="/images/fire force.jpg" width="400" height="300"> 
+        <br>     
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <h2>My projects</h2>
+    </div>
+        <p> 
+            I have created projects such as a Binary Tree and 
+            and Information System projects that involve utilizing Access and Excel
+            to analyze statistical data. The list of my work is on GitHub. However, especially
+            still being at an entry level, I haven't created very many projects since the 
+            Computer IT major does not focus on creating programs, rather you are handling 
+            pre-built systems and operating systems overall. But I do hope to create more,
+            especially with this opportunity of the summer!
+        </p2>  
+        <br>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <h2>Fun Fact About Me</h2>
+    </div>
+        <p>The Game that I like the most is Star Wars Battlefront 2</p>
+    </div>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <h2>Send Me Your Email</h2>
+    </div>
+      <form action="/emailReceiver" method="POST">
+        <p>Enter your email:</p>
+        <input name="email-input" placeholder="Ex. email@gmail.com">EMAIL</input>
```

Comment:
I don't think you need to have `EMAIL` between the input tags if you already have a placeholder.